### ISSUE TITLE
NativeWind v4 - Coming soon 🎉🎉🎉

### ISSUE THREAD

**Issue by marklawlor** [nativewind/nativewind#600]
Hi everyone,

NativeWind v4 is coming soon and is available on npm for people to try :tada: 

> [!WARNING]  
> This is not a stable release of v4 - you may encounter bugs and/or other issues. Documentation is not finalised.

Announcement (WIP, missing some details): https://www.nativewind.dev/v4/announcement
Documentation: https://www.nativewind.dev/v4/overview

**Comment by marklawlor** [comment]
Hi everyone, 

We accidentally published v4 as `latest` on NPM which has created some confusion.

While v4 is almost ready, it does have breaking changes and users should take consideration when upgrading. We're still working on the announcement and upgrade guide, which is why we have rolled back the `latest` label to v2. 

> [!NOTE]  
> If you have the error `.plugins is not a valid Plugin property`, you are using v4 with a v2 setup.

v4 is almost ready - we're just waiting on feedback from users before its released. If you'd like to upgrade you can do so using the links above.

I'm sorry this occurred, I've put in safe guards to ensure it doesn't happen again.

**Comment by danstepanov** [comment]
📢 v4.1 Update

Just one more test left to fix…

**Comment by danstepanov** [comment]
📢v4.1 Update

🎉 All tests are passing! 🎉

Last thing to do is to fix animation switching. Animations work on their own; however, switching and/or changing them causes things to not work/crash.

We’ll start adding people to the Discord today for testing this week.

https://x.com/nativewindcss/status/1823061543283720631?s=46&t=ktI223rpD8Wzr88UqL4JQg

**Comment by danstepanov** [comment]
📢 v4.1 Update

Removing one polyfill today/tomorrow. Then just writing up some release notes. 

Canary release this week!

Discord is live
https://discord.gg/MqrnNWNj

https://x.com/nativewindcss/status/1826134088941543928?s=46&t=ktI223rpD8Wzr88UqL4JQg

**Comment by danstepanov** [comment]
📢 v4.1 Update

We're aiming to merge `next` to `main` on Monday, September 2.

In anticipation of this, I'm closing all of the existing issues. However, I want to be mindful of the potential relevance of these issues.

As such, I've marked all of these issues with the `issue-wipe-pre-v4` label. The description of this label provides more detail as to how you can resurface a issue that persists post the merge of v4 into the `main` branch:

> We are closing all pre-v4 issues now that v4 is released. If the problem is still happening with v4, please open a new issue and reference this one in the description.

I will be taking a similar approach with the existing pull requests. Given the time since the last merge to `main` (in 2022), this is the most effective way to get the Github back on track with timely updates.

As always, we are incredibly grateful to everyone who has intended to contribute to NativeWind. We're looking forward to your contributions moving forward. I am actively working to make this process easier with the contribution guide.
